core/controllers/library.py

```diff
@@ -143,6 +148,83 @@ def get(self):
         self.render_json(self.values)
 
 
+class LibraryGroupPage(base.BaseHandler):
+    """The page for displaying top rated and recently published explorations.
+    """
+
+    def get(self):
+        """Handles GET requests."""
+        group_mode = (feconf.LIBRARY_GROUP_RECENTLY_PUBLISHED or
```

Comment:
Yeah I agree. I will set group_mode = True. Just introduced the check to be on the safe side.